sys/geom/label/g_label_swaplinux.c

```diff
@@ -0,0 +1,111 @@
+/*-
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright (c) 2024 Ricardo Branco
+ * All rights reserved.
```

Comment:
we've been trying our hardest to get rid of this line

Reply:
> we've been trying our hardest to get rid of this line

I copied it from the other files in the folder.

Reply:
ordinarily, I  would say something like: "yeah, but that probably means we weren't able to track down the original author and ask them if we can remove that line"

but @pjd is still a very active committer, so, 🤷🏻‍♀️

Reply:
It takes a concerted effort to find all the people with all rights reseved lines, ask them then remove. It gets more complicated when there are multiple copyright holders. And it's not the #1 priority

Reply:
Hopefully fixed now.